src/attributes/attr.js

```diff
@@ -3,11 +3,14 @@ define([
 	"../core/access",
 	"./support",
 	"../var/rnotwhite",
+	"./val",
```

Comment:
It doesn't exist in current `compat`: https://github.com/jquery/jquery/blob/d0388e9e806ca3b30e7bbaaa1c336b7c98dc5f88/src/attributes/attr.js#L6